Configurations/unix-Makefile.tmpl

```diff
@@ -1146,7 +1226,8 @@ EOF
       $recipe .= <<"EOF";
 $full: $deps
 	\$(CC) \$(LIB_CFLAGS) $linkflags\$(LIB_LDFLAGS)$shared_soname$shared_imp \\
-		-o $full$shared_def $objs \\
+		-o $full$shared_def \\
+		$objs \\
                 $linklibs \$(LIB_EX_LIBS)
```

Comment:
Indentation here seems strange

Reply:
So github has started showing tabs as 4 spaces.  Great...
(tabs aren't stricly necessary in continuation lines, but uniformity is nice, so I'll fix)

Reply:
> So github has started showing tabs as 4 spaces. Great...

I don't like conspiracy theories, but Windows (esp DevStudio) treats tab as four spaces.  The MSFT assimilation is complete.